src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/RightCurlyCheck.java

```diff
@@ -141,6 +143,7 @@ public void setShouldStartLine(boolean flag) {
             TokenTypes.LITERAL_FINALLY,
             TokenTypes.LITERAL_IF,
             TokenTypes.LITERAL_ELSE,
+            TokenTypes.LAMBDA,
```

Comment:
@romani Not all acceptable tokens in this check are default.
It is ok to have `lambda` here?
I believe you said we add default tokens when token won't do much damage. How did we decide this for lambda in this check?